blog.md

```diff
@@ -0,0 +1,203 @@
+# CuTropicalGEMM.jl: A fast package for Matrix Multiplication of Tropical Numbers on GPU
```

Comment:
```suggestion
# CuTropicalGEMM.jl: The fasted package for Tropical Matrix Multiplication on GPU
```

Reply:
Or we can have a more sexy title:
"How to implement generic matrix multiplication (GEMM) with generic element types on GPU"